common/presenters/framework-step-to-summary.js

```diff
@@ -1,15 +1,30 @@
-const { filter, flatten } = require('lodash')
+const { filter, find, flatten, isArray, isObject } = require('lodash')
 
 const frameworksHelpers = require('../helpers/frameworks')
 
 const frameworkFieldToSummaryListRow = require('./framework-field-summary-list-row')
 
 function frameworkStepToSummary(allFields, responses, baseUrl) {
-  return ([key, step]) => {
+  return ([key, step], i, allSteps = []) => {
+    const unmetStepConditions = allSteps
+      .map(step => step[1].next)
+      .filter(next => isArray(next))
+      .filter(next => {
+        const conditions = next
+          .filter(condition => isObject(condition))
+          .filter(condition => condition.next === step.slug)
+          .filter(item => {
+            const response = find(responses, ['question.key', item.field]) || {}
+            return item.value !== response.value
+          })
+
+        return conditions.length
+      })
+
     const stepUrl = `${baseUrl}/${step.slug}`
     const stepFields = step.fields
 
-    if (!stepFields.length) {
+    if (!stepFields.length || unmetStepConditions.length) {
```

Comment:
`.length` is a number, but you're using it as a truthy/falsy value. I'd rather have `stepFields.length !== 0 || unmetStepConditions.length > 0`

Reply:
Late night coding fail...